app/controllers/api/v1/packages_controller.rb

```diff
@@ -14,7 +16,7 @@ class PackagesController < Api::V1::ApiController
 
     def_param_group :package do
       param :package, Hash, required: true do
-        param :quantity, Integer, desc: "Package quantity", allow_nil: true
+        param :quantity, lambda { |val| [String, Fixnum].include? val.class }, desc: "Package quantity", allow_nil: true
```

Comment:
Interesting... I see '.json' is missing from packages... perhaps that affects how the values are parameterized. I knew Rails used to do it but I thought they stopped as things like symbols (which live forever) could cause denial of service issues when you run out of memory.